src/components/Navigation.astro

```diff
@@ -0,0 +1,126 @@
+<nav class="main-nav" id="#Home">
+  <img src="public/img/Logo_black.svg" alt="" class="logo" />
```

Comment:
you should change the src path accoring to astro 
`Instead of /public/img/Logo_black.svg, use /img/Logo_black.svg. (x42)`
![image](https://github.com/karczujr/Zasun.eu---Astro/assets/27014858/44b5daf5-57d2-4975-a11a-4daedcc278a8)